lib/api/apiUtils/object/BackendInfo.js

```diff
@@ -0,0 +1,24 @@
+import config from '../../../Config';
+
+export class BackendInfo {
```

Comment:
As discussed, let's put this function in its own file in the utilities directory.